Cargo.toml

```diff
@@ -129,7 +129,7 @@ etcd-client = "0.14"
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "a7274ddce299f33d23dbe8af5bbe6219f07c559a" }
+greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "aa706388a42c12d9294d3723113ef37169dd3de8" }
```

Comment:
Update this after https://github.com/GreptimeTeam/greptime-proto/pull/225 is merged